scheduler: 
  {
    begin_action: 1,
    parking_spot: 0.63,
    parking_steering_mode: 1,
    starting_distance: 1.0,
    num_park_to_complete: 2
  }
